app/views/customers/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>First name:</strong>
+  <%= @customer.first_name %>
+</p>
+
+<p>
+  <strong>Last name:</strong>
+  <%= @customer.last_name %>
+</p>
+
+<p>
+  <strong>Phone:</strong>
+  <%= @customer.phone %>
+</p>
+
+<p>
+  <strong>Email:</strong>
+  <%= @customer.email %>
+</p>
+
+<%= link_to 'Edit', edit_customer_path(@customer) %> |
+<%= link_to 'Back', customers_path %>
+
+  <!-- if that customer has orders, a list of them are shown in a table at the bottom, 
+  with show/update/delete links. 
+   You will need to add the appropriate has_many line to the customer model. -->
+
+  <h1>Orders</h1>
+
+<table>
```

Comment:
If the customer has no Orders, the table should not appear, this can be solved by adding an if condition checking if customer has orders before the h1 tag.